### PR TITLE
Ensure consistent field initialization

### PR DESCRIPTION
**Pull request by AliceYuan**

### CODE REVIEW

File: client/scraper/error_metadata.py [hunk 1]
```diff
@@ -9,6 +9,14 @@
 
 class ErrorMetadata(Metadata):
 
+    def init_fields(self):
+        self.prop_map[FieldKeyword.STATUS] = None
+        self.prop_map[FieldKeyword.URL] = None
+        self.prop_map[FieldKeyword.REQUEST_URL] = None
+        self.prop_map[FieldKeyword.PROVIDER_URL] = None
+        self.prop_map[FieldKeyword.TITLE] = None
```
Comment (MeenuV) [inline client/scraper/error_metadata.py]: I don't think we should have title in this. I find it misleading to have a title for a page that has errored out

**Comment by MeenuV** [comment]
+1 with title change

File: client/scraper/error_metadata.py (hunk 1 above)
Comment (AliceYuan) [inline client/scraper/error_metadata.py]: but shouldn't we give more information than less? actually description should be here too.
The developer should choose what info to show

Comment (MeenuV) [inline client/scraper/error_metadata.py]: Hmm, ok, I concede

**Pull request merged**